### DISCUSSION TITLE
Missed challenge

### DISCUSSION THREAD

**Discussion by SAVAMforWIN** [Chia-Network/chia-blockchain#8123] [Support]
Hello guys,
i am getting some unusual log errors, can you help me figure out what the problem is?

![image](https://user-images.githubusercontent.com/34774419/130036302-ce40398e-7d4a-494e-b21e-d1859167f29f.png)

![image](https://user-images.githubusercontent.com/34774419/130036415-003284f7-8448-4463-8ed4-47aa7c2d0896.png)

![image](https://user-images.githubusercontent.com/34774419/130036477-4ca49c5e-ec73-42e5-8310-d2b01c35880f.png)

**Comment by BrandtH22** [comment]
@SAVAMforWIN , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - it is generally recommended to run a chia plots check on any plots that have errors or warnings presented in the logs to ensure the plot is of good quality:
https://docs.chia.net/cli#plots-check

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
